CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.4.4]
+### Added
+- Retain tag history on update command for renamed packages with new options ([#182](https://github.com/MetaMask/auto-changelog/pull/182))
+
```

Comment:
Extra newline here:
```suggestion
```

Curious that this didn't result in a lint error :thinking:

Reply:
yes indeed it didn't any error